plotEff.cc

```diff
@@ -49,7 +49,7 @@ void plotEffBin(int q2Bin, int parity, bool doClosure)
   int distBins = 10;
 
   // Load variables and dataset
-  string filename_data = Form("effDataset_b%i.root",q2Bin);
+  string filename_data = Form("/eos/cms/store/user/fiorendi/p5prime/effKDE/%i/lmnr/effDataset_b%i_%i.root",year,q2Bin,year);
```

Comment:
same as in plotHist.cc

Reply:
addressed [here](https://github.com/CMSKStarMuMu/eff-KDE/pull/2/commits/0d1163fbc6b25f3bb7966a660170b1394346356d)